src/components/app/routes/loaders/rootLoader.ts

```diff
@@ -39,7 +39,7 @@ const makeRootLoader: Types.MakeRouteLoaderFunctionWithQueryClient = function ma
     });
 
     // User has no active, linked enterprise customer and no staff-only customer metadata exists; return early.
-    if (!enterpriseLearnerData.enterpriseCustomer) {
+    if (!enterpriseLearnerData.enterpriseCustomer && !enterpriseLearnerData.activeEnterpriseCustomer) {
       return null;
     }
```

Comment:
Ensures redirect to active ECU, if available, on BFF-enabled route vs. a 404 via `ensureActiveEnterpriseCustomerUser` below.